package.json

```diff
@@ -51,10 +51,10 @@
 		"@infinumjs/eslint-config-react-js": "^3.5.0",
```

Comment:
you need to change the version in package.json for this release

Reply:
It's there, GH didn't detect it for some reason 🤷